tests/constructor-basic/constructor-basic.mutation.test.js

```diff
@@ -0,0 +1,287 @@
+    expect(receivedConstructorBasic).toBeDefined();
+    expect(createConstructor.data.addConstructorBasic).toEqual({
+      ...currentConstructorBasic,
+      _id: constructorBasicID,
+    });
+    done();
+  });
+
+  test('#2 Should update existing constructorBasic ', async done => {
+    const beforeUpdateInput = getConstructorDataBeforeUpt(constructorInput);
+
+    const constructorIdUpt = await createConstructorBasic(beforeUpdateInput);
+
+    const updateConstructor = await operations.mutate({
+      mutation: gql`
+        mutation($id: ID!, $constructorElement: ConstructorBasicInput!) {
+          updateConstructorBasic(
+            id: $id
+            constructorElement: $constructorElement
+          ) {
+            ... on ConstructorBasic {
+              _id
+              name {
+                lang
+                value
+              }
+              material {
+                _id
+              }
+              color {
+                _id
+              }
+              image
+              available
+              default
+            }
+            ... on Error {
+              statusCode
+              message
+            }
+          }
+        }
+      `,
+      variables: {
+        constructorElement: constructorUpdateInput,
+        id: constructorIdUpt._id,
+      },
+    });
+    const constructorAfterUpdate =
+      updateConstructor.data.updateConstructorBasic;
+    construrtorIDafter = constructorAfterUpdate._id;
+    const constructorData = getConstructorDataForUptCompare(
+      constructorAfterUpdate
+    );
+    expect(constructorAfterUpdate).toBeDefined();
+    expect(constructorData).toEqual({
+      ...constructorUpdateInput,
+      _id: constructorAfterUpdate._id,
+    });
+    done();
+  });
+
+  test('#3 ConstructorBasic should return Error ConstructorBasic already exist', async done => {
+    const createConstructorF = await operations.mutate({
+      mutation: gql`
+        mutation($constructorElement: ConstructorBasicInput!) {
+          addConstructorBasic(constructorElement: $constructorElement) {
+            ... on ConstructorBasic {
+              _id
+            }
+            ... on Error {
+              statusCode
+              message
+            }
+          }
+        }
+      `,
+      variables: { constructorElement: constructorInput },
+    });
+    const createConstructor = await operations.mutate({
+      mutation: gql`
+        mutation($constructorElement: ConstructorBasicInput!) {
+          addConstructorBasic(constructorElement: $constructorElement) {
+            ... on ConstructorBasic {
+              _id
+            }
+            ... on Error {
+              statusCode
+              message
+            }
+          }
+        }
+      `,
+      variables: { constructorElement: constructorInput },
+    });
+
+    const error = createConstructor.data.addConstructorBasic.message;
+    expect(error).toBeDefined();
+    expect(error).toEqual('BASIC_ALREADY_EXIST');
```

Comment:
I think? it would be better to import `BASIC_ALREADY_EXIST` error message from separate config file.

Reply:
added import BASIC_ALREADY_EXIST error message from separate config file